src/pages/publication/references_tab/Journal.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { PublicationChannel } from './../../../types/references.types';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import CancelIcon from '@material-ui/icons/Cancel';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { Button } from '@material-ui/core';
+
+const StyledJournal = styled.div`
+  margin-top: 0.5rem;
+  border-color: black;
+  border-width: 1pt;
+  background-color: ${({ theme }) => theme.palette.background.default};
+  display: grid;
+  grid-column-gap: 0.5rem;
+  align-items: center;
+  grid-template-areas:
+    'icon issnLabel levelLabel publisherLabel button'
+    'icon issn level publisher button';
+  grid-template-columns: 5% 40% 10% 30% 10%;
+`;
+
+const StyledJournalLabel = styled.div`
+  font-size: 0.7rem;
+  font-weight: bold;
+`;
+
+const StyledIssnLabel = styled(StyledJournalLabel)`
+  grid-area: issnLabel;
+`;
+
+const StyledLevelLabel = styled(StyledJournalLabel)`
+  grid-area: levelLabel;
+`;
+
+const StyledPublisherLabel = styled(StyledJournalLabel)`
+  grid-area: publisherLabel;
+`;
+
+const StyledIssnText = styled.div`
+  grid-area: issn;
+`;
+
+const StyledLevelText = styled.div`
+  grid-area: level;
+`;
+
+const StyledPublisherText = styled.div`
+  grid-area: publisher;
+`;
+
+const StyledCheckCircleIcon = styled(CheckCircleIcon)`
+  grid-area: icon;
+  color: green;
+  margin: 0.5rem;
+  font-size: 2rem;
+`;
+
+const StyledCancelIcon = styled(CancelIcon)`
+  grid-area: icon;
+  color: red;
+  margin: 0.5rem;
+  font-size: 2rem;
+`;
+
+const StyledButton = styled(Button)`
+  grid-area: button;
+  background-color: red;
+  color: white;
+`;
+
+interface JournalProps {
+  journal: PublicationChannel;
+  setValue: (value: string) => void;
+}
+
+const Journal: React.FC<JournalProps> = ({ journal, setValue }) => {
+  const { t } = useTranslation('publication');
+
+  return (
```

Comment:
trengs det to ( her?
enig med @elkebab om at det er dumt å rendre en div som ikke brukes til noe